mmv1/products/compute/Disk.yaml

```diff
@@ -431,6 +432,10 @@ properties:
       Indicates how many IOPS must be provisioned for the disk.
     required: false
     default_from_api: true
+    description: |
+     Update currently only supported by hyperdisk skus, allowing for an update of IOPS every 4 hours
+    update_verb: :PATCH
```

Comment:
Does adding an update_verb on a resource marked immutable work? Were you able to successfully update in-place a disk with a provider built off of this?

Reply:
It works I have tested it successfully I have built the provider successfully and it updates inplace no issue, while maintaining the ForceNew on all other PDs except Hyper Disk